### PR TITLE
Exposing oauth-authorization-server, too

### PR DESCRIPTION
**Pull request by mitar**
See [this GitLab issue](https://gitlab.com/gitlab-org/gitlab/-/issues/233956) for more background.

In short, both of those URIs and compatible to each other and contain a similar document with possibly slightly different fields, but both allow other fields to be present. So I propose we expose the same document on both locations.

Current document is already a mix of both specs, for example, `revocation_endpoint` is in fact defined for `oauth-authorization-server` and not `openid-configuration`, but it is exposed under `openid-configuration`. Which is fine.

So, in short, for maximum compatibility, let's expose this on both paths.

### CODE REVIEW

**Comment by mitar** [comment]
Any update here?

**Review by nbulaj — APPROVED** [review]
:+1: LGTM

Could you please add a note to the changelog? Thanks!

**Comment by nbulaj** [comment]
I'll do it myself, thanks

**Comment by mitar** [comment]
Thanks! Sorry, I was busy and could not get to it immediately. Thank you for picking it up.
